lib/html-min.cls.php

```diff
@@ -166,11 +170,28 @@ public function process()
 		return $this->_html;
 	}
 
+	/**
+	 * From LSCWP 6.2: Changed the function to test for special comments that will be skipped. See: https://github.com/litespeedtech/lscache_wp/pull/622
+	 */
 	protected function _commentCB($m)
 	{
-		return (0 === strpos($m[1], '[') || false !== strpos($m[1], '<!['))
-			? $m[0]
-			: '';
+		// Check if is IE conditional comment
+		$is_ie_comment = (0 === strpos($m[1], '[') || false !== strpos($m[1], '<!['));
+		if($is_ie_comment) return $m[0];
+
+		// Check if comment text is present in Page Optimization -> HTML Settings -> HTML Keep comments
+		$is_skip_comment = false;
+		if(count($this->_skipComments) > 0){
+			foreach ($this->_skipComments as $comment) {
+				if ($comment && strpos($m[1], $comment) !== false) {
+					$is_skip_comment = true;
```

Comment:
No need to use $is_skip_comment. Just return `$m[0]`. This way you can even save the next line `break;`

Reply:
Done